src/ui/components/nav/components/SearchBar.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
```

Comment:
I think this doesn't need to be a client component. You can use a server component with [server actions](https://nextjs.org/docs/app/api-reference/functions/server-actions) and call [`redirect`](https://nextjs.org/docs/app/api-reference/functions/redirect)

Reply:
@aidensgithub yes, it's more idiomatic in this context (App Router)